src/routes/email/entities/delete-email-dto.entity.ts

```diff
@@ -0,0 +1,9 @@
+import { ApiProperty } from '@nestjs/swagger';
+
+export class DeleteEmailDto {
+  @ApiProperty()
+  account: string;
+
+  @ApiProperty()
+  emailAddress: string;
```

Comment:
the `emailAddress` shouldn't be required.

Reply:
Removed in 27eaae4e907918ecba4248fe5b52aceb7c32a10e.